tests/subsys/fs/fat_fs_api/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
   filesystem.fat:
-    platform_whitelist: arduino_101 native_posix
+    platform_whitelist: native_posix
```

Comment:
The commit message says you're enabling "broader testing on all boards with needed
features", so why are you still whitelisting?

Reply:
yeah, we going through commits adding bodies almost like a robot. Will change.

Reply:
removed this commit from the series for now.